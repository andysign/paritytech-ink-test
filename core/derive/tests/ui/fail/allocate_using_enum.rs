use ink_core_derive::AllocateUsing;

#[derive(AllocateUsing, Debug, PartialEq, Eq)]
enum Enum {
    A,
    B,
    C,
}

fn main() {}
