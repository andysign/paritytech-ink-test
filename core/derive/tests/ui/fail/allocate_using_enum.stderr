error: proc-macro derive panicked
 --> $DIR/allocate_using_enum.rs:3:10
  |
3 | #[derive(AllocateUsing, Debug, PartialEq, Eq)]
  |          ^^^^^^^^^^^^^
  |
  = help: message: cannot derive AllocateUsing for enums
